release-notes/10.0/preview/preview3/aspnetcore.md

```diff
@@ -2,14 +2,271 @@
 
 Here's a summary of what's new in ASP.NET Core in this preview release:
 
-- [Feature](#feature)
+- [Declarative model for persisting state from components and services](#declarative-model-for-persisting-state-from-components-and-services)
+- [Reference fingerprinted static web assets in standalone Blazor WebAssembly apps](#reference-fingerprinted-static-web-assets-in-standalone-blazor-webassembly-apps)
+- [`HttpClient` response streaming enabled by default on WebAssembly](#httpclient-response-streaming-enabled-by-default-on-webassembly)
+- [`DisableMatchAllIgnoresLeftUriPart` app context switch renamed to `EnableMatchAllForQueryStringAndFragment`](#disablematchallignoreslefturipart-app-context-switch-renamed-to-enablementchallofthequerystringandfragment)
+- [Set the environment at build-time for standalone Blazor WebAssembly apps](#set-the-environment-at-build-time-for-standalone-blazor-webassembly-apps)
+- [Validation support in minimal APIs](#validation-support-in-minimal-apis)
+- [OpenAPI support enabled by default in the ASP.NET Core Web API (native AOT) template](#openapi-support-enabled-by-default-in-the-aspnet-core-web-api-native-aot-template)
+- [Support for Server-Sent Events (SSE)](#support-for-server-sent-events-sse)
 
 ASP.NET Core updates in .NET 10:
 
 - [What's new in ASP.NET Core in .NET 10](https://learn.microsoft.com/aspnet/core/release-notes/aspnetcore-10.0) documentation.
 - [Breaking changes](https://docs.microsoft.com/dotnet/core/compatibility/10.0#aspnet-core)
 - [Roadmap](https://github.com/dotnet/aspnetcore/issues/59443)
 
-## Feature
+## Declarative model for persisting state from components and services
+
+You can now declaratively specify state to persist from components and services using the `SupplyParameterFromPersistentComponentState` attribute. Properties with this attribute will automatically be persisted using the `PersistentComponentState` service during prerendering and then loaded when the component renders interactively or the service is instantiated.
```

Comment:
```suggestion
You can now declaratively specify state to persist from components and services using the `SupplyParameterFromPersistentComponentState` attribute. Properties with this attribute will automatically be persisted using the `PersistentComponentState` service during prerendering and then load when the component renders interactively, or the service instantiates.
```

Reply:
The original version sounds more correctly to me.